src/util.h

```diff
@@ -36,6 +36,7 @@ int64_t GetStartupTime();
 static const bool DEFAULT_LOGTIMEMICROS = false;
 static const bool DEFAULT_LOGIPS        = false;
 static const bool DEFAULT_LOGTIMESTAMPS = true;
+extern const char * const DEFAULT_DEBUGLOGFILE;
```

Comment:
Would have expected this to have accompanied

    extern const char * const BITCOIN_CONF_FILENAME;
    extern const char * const BITCOIN_PID_FILENAME;

lower in the file.

Reply:
I thought about that, but it's grouped with the other `DEFAULT_` here, that makes more sense to me than grouping by type.

Reply:
The others are defaults that can be overridden via an option; calling it BITCOIN_DEBUGLOG_FILENAME could make it fit by both name and type?

Reply:
I don't get you. This is the default for `-debuglogfile`, which can is an option. We use the convention `DEFAULT_X` where X is the option name everywhere.

Reply:
Same thing is trust for `BITCOIN_{CONF,PID}_FILENAME` - they're constants that can't be changed and are only used for getArg and HelpMessageOpt. But that's a better argument for renaming those to `DEFAULT_X` like almost every single other option, so nevermind.

Reply:
I agree! the DEFAULT_ convention is more frequently used so would make sense to use everywhere. BITCOIN_CONF_FILENAME and BITCOIN_PID_FILENAME are very old options so they didn't follow that yet. Feel free to send a patch (but not to this PR as it's pretty out of scope :-)